pom.xml

```diff
@@ -0,0 +1,258 @@
+		<version>2.2.5.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+
+	<groupId>org.gridsuite</groupId>
+	<artifactId>gridsuite-gateway</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+
+	<name>gridsuite gateway</name>
+	<description>Entry point for backend services</description>
+
+	<scm>
+		<connection>scm:git:https://github.com/gridsuite/gateway.git</connection>
+		<developerConnection>scm:git:https://github.com/gridsuite/gateway.git</developerConnection>
+		<url>https://github.com/gridsuite/gateway</url>
+	</scm>
+
+	<developers>
+		<developer>
+			<name>Chamseddine BENHAMED</name>
+			<email>chamseddine.benhamed@rte-france.com</email>
+			<organization>RTE</organization>
+			<organizationUrl>http://www.rte-france.com</organizationUrl>
+		</developer>
+	</developers>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<java.version>11</java.version>
+
+		<maven.clean.version>3.1.0</maven.clean.version>
+		<maven.checkstyle.version>3.0.0</maven.checkstyle.version>
+		<maven.compiler.version>3.8.0</maven.compiler.version>
+		<maven.install.version>2.5.2</maven.install.version>
+		<maven.jacoco.version>0.8.2</maven.jacoco.version>
+		<maven.jib.version>1.8.0</maven.jib.version>
+		<maven.resources.version>3.1.0</maven.resources.version>
+		<maven.site.version>3.7.1</maven.site.version>
+		<maven.surefire.version>2.22.1</maven.surefire.version>
+
+		<guava.version>20.0</guava.version>
+		<datastax.version>3.7.2</datastax.version>
+		<embedded-cassandra.version>3.0.3</embedded-cassandra.version>
+		<lombok.version>1.18.8</lombok.version>
+		<metricscore.version>3.2.6</metricscore.version>
+		<springboot.version>2.2.2.RELEASE</springboot.version>
+		<springdata.version>2.2.3.RELEASE</springdata.version>
+		<springfox.version>2.9.2</springfox.version>
+		<springtest.version>5.1.5.RELEASE</springtest.version>
+
+
+		<spring-cloud.version>Hoxton.SR3</spring-cloud.version>
+		<integration.skip>true</integration.skip>
+	</properties>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dependencies</artifactId>
+				<version>${spring-cloud.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
+	<dependencies>
+		<dependency>
+			<groupId>io.jsonwebtoken</groupId>
+			<artifactId>jjwt-api</artifactId>
+			<version>0.11.1</version>
+		</dependency>
+		<dependency>
+			<groupId>io.jsonwebtoken</groupId>
+			<artifactId>jjwt-impl</artifactId>
+			<version>0.11.1</version>
+			<scope>runtime</scope>
+		</dependency>
+		<dependency>
+			<groupId>io.jsonwebtoken</groupId>
+			<artifactId>jjwt-jackson</artifactId>
+			<version>0.11.1</version>
+			<scope>runtime</scope>
+		</dependency>
+		<dependency>
+			<groupId>io.jsonwebtoken</groupId>
+			<artifactId>jjwt</artifactId>
+			<version>0.9.1</version>
```

Comment:
So we have 3 jwt libraries: io.jsonwebtoken:jjwt, com.auth0:java-jwt and com.nimbusds:nimbus-jose-jwt
We should keep only 1